cmd/zt_remove_file_test.go

```diff
@@ -76,14 +76,19 @@ func (s *cmdIntegrationSuite) TestRemoveFilesUnderShare(c *chk.C) {
 	raw := getDefaultRemoveRawInput(rawShareURLWithSAS.String())
 	raw.recursive = true
 
+	includeRootInTransfers := true // this is our current behaviour (schedule it, but STE does nothing for that removal transfer).
+	// TODO: review
+
```

Comment:
why wouldn't STE remove the root? It seems counter-intuitive to recursively delete a folder but not actually delete the folder.

Unless this is the container root

Reply:
I have clarified the comment, thanks.

Enumerators always include the root.  If the root is a directory, then the STE will delete it. But if the root is the share itself (container root as per your comment) then the STE will skip it.